libs/richtext/src/lib/to-html.tsx

```diff
@@ -0,0 +1,30 @@
+import {Node} from 'slate'
+import {slateToDomConfig, slateToHtml} from 'slate-serializers'
+import {BlockFormat} from './block-format'
+import {InlineFormat} from './inline-format'
+import {TextFormat} from './text-format'
+
+export const toHtml = (nodes: Node[]) =>
```

Comment:
@antkiewiczk the HTML to Slate could probably also need something like this, as we currently only rewrite `p` to `paragraph`.